src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -4,7 +4,8 @@ import PropTypes from 'prop-types';
 class ConfirmationBox extends React.Component {
 	handleYes = () => {
 		this.props.setActualItem(this.props.name);
-		this.props.history.push('/slackname');
+		const nextPage = this.props.sendWithPhoto ? 'snackchat' : 'slackname';
```

Comment:
This is probably one of those places where destructuring might look nicer
e.g. add a line

```const { setActualItem, name, sendWithPhoto, history } = this.props;```

then remove all the ```this.props``` from elsewhere